src/index.js

```diff
@@ -158,12 +158,23 @@ export default class RolloverTodosPlugin extends Plugin {
       // Potentially filter todos from yesterday for today
       let todosAdded = 0
       let emptiesToNotAddToTomorrow = 0
-      let todos_today = !removeEmptyTodos ? todos_yesterday : []
+      let todos_today = removeEmptyTodos ? [] : todos_yesterday 
       if (removeEmptyTodos) {
         todos_yesterday.forEach((line, i) => {
           const trimmedLine = (line || "").trim()
           if ((trimmedLine != '- [ ]') && (trimmedLine != '- [  ]')) {
-            todos_today.push(line)
+            if (preserveIndentation){
+              // Get index of first non-whitespace character
+              // (Equivalent to counting)
+              indentation = line.search(/\S|$/);
```

Comment:
This will treat tabs and spaces the same I think (`"\thello".search(/\S|$/) === " hello".search(/\S|$/)` yields `true`), however you convert all these "indents" to spaces later on.
As a user, I'd expect my tabs not to be changed to spaces by the plugin :slightly_smiling_face: 

Maybe a more straightforward way of achieving what you want to do would be to simply trim the end of `line`? Why are we trimming these lines in the first place?